core/src/main/resources/hudson/PluginManager/_table.js

```diff
@@ -10,46 +10,214 @@ function checkPluginsWithoutWarnings() {
+                                htmlDivElement.classList.add('alert', 'alert-danger');
+                                htmlDivElement.innerHTML = plugin.newerJavaRequired
+                                htmlTableDataCellElement.append(htmlDivElement)
+                            }
+
+                            if (plugin.dependenciesNewerJava) {
+                                htmlDivElement = document.createElement('div');
+                                htmlDivElement.classList.add('alert', 'alert-danger');
+                                htmlDivElement.innerHTML = plugin.dependenciesNewerJava
+                                htmlTableDataCellElement.append(htmlDivElement)
+                            }
+
+                            if (plugin.unresolvedSecurityWarnings) {
+                                htmlDivElement = document.createElement('div');
+                                htmlDivElement.classList.add('alert', 'alert-danger');
+                                htmlDivElement.innerText = plugin.unresolvedSecurityWarnings.text
+                                var listElement = document.createElement('ul');
+                                plugin.unresolvedSecurityWarnings.warnings.forEach(function(warning) {
+                                    var li = document.createElement('li');
+                                    var htmlAnchorElement = document.createElement('a');
+                                    htmlAnchorElement.href = warning.url
+                                    htmlAnchorElement.text = warning.message
+                                    htmlAnchorElement.target = '_blank'
+                                    htmlAnchorElement.rel = 'noopener noreferrer'
+                                    li.append(htmlAnchorElement)
+                                    listElement.append(li)
+                                })
+                                htmlDivElement.append(listElement)
+                                htmlTableDataCellElement.append(htmlDivElement)
+                            }
+
+                            if (plugin.deprecated) {
+                                htmlDivElement = document.createElement('div');
+                                htmlDivElement.classList.add('alert', 'alert-warning');
+                                htmlDivElement.innerHTML = plugin.deprecated
+                                htmlTableDataCellElement.append(htmlDivElement)
+                            }
+
+                            if (plugin.adoptMe) {
+                                htmlDivElement = document.createElement('div');
+                                htmlDivElement.classList.add('alert', 'alert-warning');
+                                htmlDivElement.innerHTML = plugin.adoptMe
+                                htmlTableDataCellElement.append(htmlDivElement)
+                            }
+
+                            if (plugin.newerVersionAvailableNotOffered) {
```

Comment:
I think I got all of the relevant warnings, some of them are only on the updates page